ts/session/sending/MessageQueue.ts

```diff
@@ -208,15 +233,21 @@ export class MessageQueue {
         (message as any).syncTarget?.length > 0
       ) {
         window.log.warn('Processing sync message');
+        window.log.warn('Sending message non-durably');
+        return await this.sendToPubKeyNonDurably(device, message as ClosedGroupNewMessage)
       } else {
         window.log.warn('Dropping message in process() to be sent to ourself');
         return;
       }
     }
 
+    if (message instanceof ClosedGroupNewMessage) {
+      return await this.sendToPubKeyNonDurably(device, message as ClosedGroupNewMessage)
+    } else {
```

Comment:
we want to make the send non durably calls explicit when they are used. So instead of using it there,
call directly sendNonDurably when you create your newClosedgroupMessage

```
const message = new ClosedGroupNewMessage(messageParams);
 return getMessageQueue().sendNonDurably(PubKey.cast(m), message);
```

like in `closedGroup.ts` and `ts/session/group/index.ts`

After you've done this, you won't need all of that code.